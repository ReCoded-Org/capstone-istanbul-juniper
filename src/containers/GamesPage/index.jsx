import React from "react";
import ProgressSection from "../../components/gamespage/GamesPageProgressSection";
import GamesPageImageGrid from "../../components/gamespage/GamesPageImageGrid";
import BottomCurve from "../../components/BottomCurve";

const GamesPage = () => {
  return (
    <>
      <ProgressSection />
      <BottomCurve />
      <GamesPageImageGrid />
    </>
  );
};

export default GamesPage;
