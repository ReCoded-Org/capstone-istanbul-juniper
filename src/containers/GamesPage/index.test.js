import React from "react";
import renderer from "react-test-renderer";
import { BrowserRouter } from "react-router-dom";
import GamesPage from ".";

describe("GamesPage container", () => {
  it("matches the tree", () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <GamesPage />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
