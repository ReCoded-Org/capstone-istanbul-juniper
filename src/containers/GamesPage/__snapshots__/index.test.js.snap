// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GamesPage Component matches the tree 1`] = `
Array [
  <div
    className="progressSectionContainer"
  >
    <div
      className="ant-row ant-row-center"
      style={Object {}}
    >
      <h3
        className="progressSectionContainer___xp"
      >
        gamesPage.experiencePoint
      </h3>
    </div>
    <div
      className="ant-row ant-row-center progressSectionContainer___row"
      style={Object {}}
    >
      <div
        className="ant-col ant-col-xs-6 ant-col-md-push-1 ant-col-lg-push-2"
        style={Object {}}
      >
        <h3
          className="progressSectionContainer___row___header progressSectionContainer___row___curLevel"
        >
          gamesPage.level
        </h3>
      </div>
      <div
        className="ant-col ant-col-xs-12"
        style={Object {}}
      >
        <div
          className="ant-progress ant-progress-line ant-progress-status-normal ant-progress-default progressSectionContainer___row___progressBar"
        >
          <div
            className="ant-progress-outer"
          >
            <div
              className="ant-progress-inner"
              style={
                Object {
                  "backgroundColor": "white",
                }
              }
            >
              <div
                className="ant-progress-bg"
                style={
                  Object {
                    "background": "#4BA747",
                    "borderRadius": "",
                    "height": 20,
                    "width": "40%",
                  }
                }
              />
            </div>
          </div>
        </div>
      </div>
      <div
        className="ant-col ant-col-xs-6 ant-col-md-pull-1 ant-col-lg-pull-2"
        style={Object {}}
      >
        <h3
          className="progressSectionContainer___row___header progressSectionContainer___row___nextLevel"
        >
          gamesPage.level
        </h3>
      </div>
    </div>
    <div
      className="ant-row progressSectionContainer___row"
      style={Object {}}
    >
      <div
        className="ant-col ant-col-3 ant-col-push-5"
        style={Object {}}
      >
        <h3
          className="progressSectionContainer___row___header"
        >
          gamesPage.previousReward
        </h3>
        <div
          className="progressSectionContainer___row___prevRewardDiv"
        >
          <img
            alt="World"
            className="progressSectionContainer___row___prevRewardDiv___reward"
            src="world.svg"
          />
        </div>
        <h3
          className="progressSectionContainer___row___header"
        >
          gamesPage.badgeDescription
        </h3>
      </div>
      <div
        className="ant-col ant-col-3 ant-col-push-13"
        style={Object {}}
      >
        <h3
          className="progressSectionContainer___row___header"
        >
          gamesPage.nextReward
        </h3>
        <div
          className="progressSectionContainer___row___header___questionMark"
        >
          <span
            aria-label="question"
            className="anticon anticon-question"
            role="img"
          >
            <svg
              aria-hidden="true"
              className=""
              data-icon="question"
              fill="currentColor"
              focusable="false"
              height="1em"
              viewBox="64 64 896 896"
              width="1em"
            >
              <path
                d="M764 280.9c-14-30.6-33.9-58.1-59.3-81.6C653.1 151.4 584.6 125 512 125s-141.1 26.4-192.7 74.2c-25.4 23.6-45.3 51-59.3 81.7-14.6 32-22 65.9-22 100.9v27c0 6.2 5 11.2 11.2 11.2h54c6.2 0 11.2-5 11.2-11.2v-27c0-99.5 88.6-180.4 197.6-180.4s197.6 80.9 197.6 180.4c0 40.8-14.5 79.2-42 111.2-27.2 31.7-65.6 54.4-108.1 64-24.3 5.5-46.2 19.2-61.7 38.8a110.85 110.85 0 00-23.9 68.6v31.4c0 6.2 5 11.2 11.2 11.2h54c6.2 0 11.2-5 11.2-11.2v-31.4c0-15.7 10.9-29.5 26-32.9 58.4-13.2 111.4-44.7 149.3-88.7 19.1-22.3 34-47.1 44.3-74 10.7-27.9 16.1-57.2 16.1-87 0-35-7.4-69-22-100.9zM512 787c-30.9 0-56 25.1-56 56s25.1 56 56 56 56-25.1 56-56-25.1-56-56-56z"
              />
            </svg>
          </span>
        </div>
        <h3
          className="progressSectionContainer___row___header"
        >
          gamesPage.unknown
        </h3>
      </div>
    </div>
  </div>,
  <div
    className="bottomCurveContainer"
  >
    <div
      className="bottomCurveContainer__bottomCurve"
    >
      <div
        className="bottomCurveContainer__bottomCurve__topHalf"
      />
      <div
        className="bottomCurveContainer__bottomCurve__bottomHalf"
      />
    </div>
  </div>,
  <div
    className="gamePageImageGrid"
  >
    <h1
      className="gamePageImageGrid__title"
    >
      gamesPage.title
    </h1>
    <div
      className="ant-row ant-row-center"
      style={Object {}}
    >
      <div
        className="ant-col gamePageImageGrid__col ant-col-xs-20 ant-col-md-9"
        style={
          Object {
            "backgroundImage": "url(mockGTAPoster.jpg)",
          }
        }
      >
        <a
          className="gamePageImageGrid__col__redirect"
          href="/"
          onClick={[Function]}
        >
          <p
            className="gamePageImageGrid__col__redirect__text"
          >
            Grand Theft Auto  
          </p>
        </a>
      </div>
      <div
        className="ant-col gamePageImageGrid__col ant-col-xs-20 ant-col-md-9"
        style={
          Object {
            "backgroundImage": "url(mockZeldaPoster.jpg)",
          }
        }
      >
        <a
          className="gamePageImageGrid__col__redirect"
          href="/"
          onClick={[Function]}
        >
          <p
            className="gamePageImageGrid__col__redirect__text"
          >
            Zelda
          </p>
        </a>
      </div>
      <div
        className="ant-col gamePageImageGrid__col ant-col-xs-20 ant-col-md-9"
        style={
          Object {
            "backgroundImage": "url(mockDestinyPoster.jpg)",
          }
        }
      >
        <a
          className="gamePageImageGrid__col__redirect"
          href="/"
          onClick={[Function]}
        >
          <p
            className="gamePageImageGrid__col__redirect__text"
          >
            Destiny
          </p>
        </a>
      </div>
    </div>
  </div>,
]
`;
